lib/ember-qunit/test-wrapper.js

```diff
@@ -1,43 +0,0 @@
-import Ember from 'ember';
-import { getContext } from 'ember-test-helpers';
-
-export default function testWrapper(qunit /*, testName, expected, callback, async */) {
-  var callback;
-  for (var _len = arguments.length, args = new Array(_len > 1 ? _len - 1 : 0), _key = 1; _key < _len; ++_key) {
-    args[_key - 1] = arguments[_key];
-  }
-
-  function wrapper() {
-    var context = getContext();
-
-    var result = callback.apply(context, arguments);
-
-    function failTestOnPromiseRejection(reason) {
-      var message;
-      if (reason instanceof Error) {
-        message = reason.stack;
-        if (reason.message && message && message.indexOf(reason.message) < 0) {
```

Comment:
We should confirm that using QUnit's default inspection for rejected promises still gives us the message and stack when possible.  It likely does, I just want to confirm...